spec/controllers/flood_risk_engine/enrollments/steps_controller/partnership_details_spec.rb

```diff
@@ -0,0 +1,64 @@
+require "rails_helper"
+module FloodRiskEngine
+  module Enrollments
+
+    describe StepsController do
```

Comment:
given that smooth navigation backwards is important to user experience, seems very little rspec testing around what happens at different stages should user click back links or buttons